docs/src/fs.rst

```diff
@@ -340,6 +340,26 @@ API
     .. note::
         These functions are not implemented on Windows.
 
+.. c:function:: uv_fs_type uv_fs_get_type(const uv_fs_t* req)
+
+    Returns `req->fs_type`.
```

Comment:
Can you add `.. versionadded:: 1.19.0` to all of the newly added functions.

Reply:
@cjihrig done!